ContentProtection/BasicPlayReady/Program.cs

```diff
@@ -128,7 +127,7 @@ private static async Task RunAsync(ConfigWrapper config)
                         StorageConnectionString, config.StorageContainerName);
 
                     // Create an AutoResetEvent to wait for the job to finish and pass it to EventProcessor so that it can be set when a final state event is received.
-                    AutoResetEvent jobWaitingEvent = new AutoResetEvent(false);
+                    AutoResetEvent jobWaitingEvent = new(false);
```

Comment:
I see you are using some newer C# syntactic sugar - just wondering if that would cause customer confusion over being very explicit about what they are doing as before? Do we prefer this syntax?

Reply:
I find the syntax better. To avoid repeating the type, another solution is to use var jobWaitingEvent = new AutoResetEvent(... but I find the syntax less elegant.

Reply:
Good with this